libcxx/include/__locale_dir/support/bsd_like.h

```diff
@@ -75,15 +75,6 @@ inline _LIBCPP_HIDE_FROM_ABI long double __strtold(const char* __nptr, char** __
   return ::strtold_l(__nptr, __endptr, __loc);
```

Comment:
Not on this file: Since recently we also have `linux.h`, it needs to be updated.